libs/partners/chroma/pyproject.toml

```diff
@@ -19,7 +19,8 @@ disallow_untyped_defs = true
 "Release Notes" = "https://github.com/langchain-ai/langchain/releases?q=tag%3A%22langchain-chroma%3D%3D0%22&expanded=true"
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<4"
+python = ">=3.9,<4"
+langchain-standard-tests = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/standard-tests"}
 [[tool.poetry.dependencies.langchain-core]]
 version = ">=0.1.40,<0.4"
```

Comment:
need to bump up compatibility

id on document field was only added more recently

```python

    id: Optional[str] = None
    """An optional identifier for the document.

    Ideally this should be unique across the document collection and formatted
    as a UUID, but this will not be enforced.

    .. versionadded:: 0.2.11
    """
```

Reply:
Hey @eyurtsev, we just bumped up the version for langchain-core to 0.2.11